sycl/include/sycl/detail/pi.h

```diff
@@ -331,7 +332,8 @@ typedef enum {
   PI_QUEUE_INFO_DEVICE_DEFAULT = 0x1095,
   PI_QUEUE_INFO_PROPERTIES = 0x1093,
   PI_QUEUE_INFO_REFERENCE_COUNT = 0x1092,
-  PI_QUEUE_INFO_SIZE = 0x1094
+  PI_QUEUE_INFO_SIZE = 0x1094,
```

Comment:
Should we remove this then?

Reply:
It looks like this can't be removed because it corresponds to CL_QUEUE_SIZE of OpenCL backend.
This patch only adds "," at the end of this line.